PWGHF/D2H/Tasks/taskFlowCharmHadrons.cxx

```diff
@@ -386,10 +386,11 @@ struct HfTaskFlowCharmHadrons {
                       aod::BCsWithTimestamps const&)
   {
     float centrality{-1.f};
+    float occupancy = getOccupancy(collision);
```

Comment:
I think that you can define this function in `HfEventSelection` so that everyone can call it in the same way